routes/profiles.route.ts

```diff
@@ -0,0 +1,47 @@
+import { Request, Response, Router } from 'express';
+import ProfileModel from '../models/profile';
+
+const router: Router = Router();
+
+// Gets the profile of the current logged in user
+router.get('/', (req: Request, res: Response) => {
+  let id = '621592eb5d1d819799a2598d'; // TODO: change this to current user
+
+  ProfileModel.findOne({ _id: id })
+    .then((profile) => {
+      res.status(200).json(profile);
+    })
+    .catch((err) => {
+      res.status(404).json(err);
+    });
+});
+
+// Gets the profile with the given id
+router.get('/:id', async (req: Request, res: Response) => {
+  let id = req.params.id;
+
+  ProfileModel.findOne({ _id: id })
```

Comment:
For future ref: database logic should be handled in the profile controller in `/controllers`.
Note for discussion: How do we want to have a separate API model specifically called for different endpoints?

Reply:
Also noting that we want to add [OpenAPI](https://swagger.io/specification/) docs in the future